spec/mailers/club_application_mailer_spec.rb

```diff
@@ -40,8 +40,8 @@
   describe 'admin notification' do
     subject { ClubApplicationMailer.admin_notification(application) }
 
-    it { should have_subject 'hackEDU Application' }
-    it { should deliver_to 'Zach Latta <team@hackedu.us>' }
+    it { should have_subject 'Hack Club Application' }
+    it { should deliver_to 'Zach Latta <team@hackclub.io>' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.